### PR TITLE
Plot top k most-probable logits in logit lens

### PR DESCRIPTION
**Pull request by nathanneuro**
I'd like to see not just the single absolute probable token, but the top k most probable tokens.

### CODE REVIEW

**Comment by nostalgebraist** [comment]
Cool, thanks!

Could you give me edit permissions for the branch on your fork ([see here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork))?  As you anticipated in your LW message, I'd like to edit your code a bit before merging.
